lib/build.js

```diff
@@ -103,17 +103,17 @@ const build = (buildConfig = config.defaultBuildConfig, options) => {
   } else {
     util.buildTarget()
     if (config.shouldSign()) {
-      util.signApp()
-    }
-
-    if (process.platform === 'win32') {
-      // Sign only binaries for widevine sig generation.
-      // Other binaries will be done during the create_dist.
-      // Then, both are merged whenarchive for installer is created.
-      util.signWinBinaries()
-
-      if (config.brave_enable_cdm_host_verification) {
-        util.generateWidevineSigFiles()
+      if (process.platform === 'win32') {
+        // Sign only binaries for widevine sig generation.
+        // Other binaries will be done during the create_dist.
+        // Then, both are merged whenarchive for installer is created.
+        util.signWinBinaries()
+
+        if (config.brave_enable_cdm_host_verification) {
```

Comment:
This sig files generation should be go out from `shouldSign()` because sig files could be generated from un-signed binaries.

Reply:
oops, didn't realize I unintentionally changed the logic here. I'll do a quick followup